lib/utils/ConfigParser.js

```diff
@@ -152,23 +152,32 @@ class ConfigParser {
         this._capabilities = merge(this._capabilities, this._config.capabilities || defaultTo, MERGE_OPTIONS)
 
         /**
-         * run single spec file only, regardless of multiple-spec specification
+         * run only specified spec files, regardless of multiple-spec specification
+         * If spec is a file on disk, it is set as the current spec, if it is not, it
+         * is treated as a string match filter for the multiple-spec specification.
          */
         if (typeof object.spec === 'string') {
-            const specs = []
+            const specs = new Set()
```

Comment:
What is the advantages if using Set here instead of just an array?

Reply:
It was to prevent the same file from being added twice.  Could de-dupe after, unless something does that later and it's unnecessary.